src/libopensc/card-cardos.c

```diff
@@ -772,8 +929,9 @@ cardos_set_security_env(sc_card_t *card,
 			    const sc_security_env_t *env,
 			    int se_num)
 {
+	cardos_data_t* priv = (cardos_data_t*)card->drv_data;
 	sc_apdu_t apdu;
-	u8	data[6];
+	u8	data[9];
```

Comment:
clang-format suggested the following change:


```
-	u8	data[9];
+	u8 data[9];
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->